tests/e2e/specs/sale-timer-test.js

```diff
@@ -0,0 +1,41 @@
+import {
+    loginUser,
+} from '@wordpress/e2e-test-utils';
+
+import {
+    scrollToBottom,
+    isElementExists,
+} from '../utils/helpers'
+
+import { URLS } from '../utils/urls';
+
+describe( 'Sale timer test cases', () => {
+    it( 'should set sale timer', async () => {
```

Comment:
Kindly adhere to [WordPress JavaScript Coding Standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/) in terms of spacing and indentions. Write doc-block if necessary.